### PR TITLE
Register kshaminnovation.is-a.dev

### PR DESCRIPTION
**Pull request by vanilla-dd**
Register kshaminnovation.is-a.dev with TXT record pointing to dh=04d46d2ae9df39be926a8b46632e5bf2a9d714a9.

### CODE REVIEW

File: domains/kshaminnovation.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vanilla-dd",
+           "email": "119125583+vanilla-dd@users.noreply.github.com",
+           "discord": "1049748998195519518"
+        },
+    
+        "record": {
+            "TXT": "dh=04d46d2ae9df39be926a8b46632e5bf2a9d714a9"
```
Comment (CuteDog5695) [inline domains/kshaminnovation.json:9]: not the way to do discord txt verification

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_discord.link_trzz.json
Comment (CuteDog5695) [inline domains/_discord.link_trzz.json:1]: you cant have an _ in the subdomain (also link_trzz.is-a.dev doesnt exist)

File: domains/_discord.link_trzz.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vanilla-dd",
+           "email": "119125583+vanilla-dd@users.noreply.github.com",
```
Comment (CuteDog5695) [inline domains/_discord.link_trzz.json:4]: please either change your email or leave the email line blank

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_discord.link_trzz.json (hunk 2 above)
Comment (vanilla-dd) [reply domains/_discord.link_trzz.json]: done

File: domains/_discord.kshaminnovation.json
Comment (CuteDog5695) [inline domains/_discord.kshaminnovation.json:1]: kshaminnovation.is-a.dev does not exist, so you cant make a nested subdomain with it

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**